src/test/util.rs

```diff
@@ -87,17 +83,9 @@ impl TestClientBuilder {
         self
     }
 
-    /// Modify options via `TestClient::options_for_multiple_mongoses` before setting them.
-    // TODO RUST-1449 Simplify or remove this entirely.
-    pub(crate) async fn additional_options(
-        mut self,
-        options: impl Into<Option<ClientOptions>>,
-        use_multiple_mongoses: bool,
-    ) -> Self {
-        let options = options.into();
-        assert!(self.options.is_none() || options.is_none());
-        self.options =
-            Some(TestClient::options_for_multiple_mongoses(options, use_multiple_mongoses).await);
+    /// When running against a sharded topology, only use the first configured host.
+    pub(crate) fn sharded_use_first_host(mut self) -> Self {
```

Comment:
This is definitely an awkward name, I welcome suggestions.

Reply:
Maybe `use_single_mongos`? That's in line with the `use_multiple_mongoses` language from the unified tests